.github/workflows/operator-maven.yaml

```diff
@@ -57,31 +57,17 @@ jobs:
           driver: docker
       - name: 'Set up Docker Buildx'
         uses: docker/setup-buildx-action@v3
-      - name: 'Configure build environment'
-        id: build_configuration
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: 'Build Kroxylicious Operand Container Image'
         run: |
           GIT_HASH="$(git rev-parse HEAD)"
-          IMAGE_TAG="dev-git-${GIT_HASH}"
           KROXYLICIOUS_VERSION="$(mvn org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=project.version -q -DforceStdout)"
-          KROXYLICIOUS_IMAGE="quay.io/kroxylicious/kroxylicious:${IMAGE_TAG}"
-          echo "kroxylicious_version=${KROXYLICIOUS_VERSION}" >> "$GITHUB_OUTPUT"
-          echo "kroxylicious_image=${KROXYLICIOUS_IMAGE}" >> "$GITHUB_OUTPUT"
-      - name: 'Build tagged Kroxylicious Operand Docker image'
-        uses: docker/build-push-action@v6
-        with:
-          file: Dockerfile
-          context: .
-          build-args:
-            KROXYLICIOUS_VERSION=${{ steps.build_configuration.outputs.kroxylicious_version }}
-          tags: ${{ steps.build_configuration.outputs.kroxylicious_image }}
-          push: false
-          cache-from: type=gha
-          cache-to: type=gha,mode=max,compression=zstd
-          load: 'true'
+          KROXYLICIOUS_IMAGE=quay.io/kroxylicious/proxy:${KROXYLICIOUS_VERSION}-${GIT_HASH}
```

Comment:
I've used a distinctive container image just so it is clear from the logs that the tests are using the right image.